blocks/core/ff_module/ff_module-task-event/src/TaskEventContainer.rt

```diff
@@ -0,0 +1,6 @@
+<div
+    class="ff_module-task-event ff_module-task-event--{this.props.event.type}"
+    >
+    <time class="ff_module-task-event__sent">{this.props.event.sent}</time>
```

Comment:
We'll need to implement whatever logic we want to use for fuzzy date logic here as well, and it feels to me like the best place to make that decision is in the component that's actually showing the date.